silhouette/app/com/mohiva/play/silhouette/impl/providers/SocialProvider.scala

```diff
@@ -165,6 +165,14 @@ trait SocialProfile {
   def loginInfo: LoginInfo
 }
 
+trait IdentifiedSocialProfile extends SocialProfile {
```

Comment:
It should be the same as described in the [documentation](http://silhouette.mohiva.com/docs/providers#section-write-a-custom-social-profile-builder). Or is there a problem I can't see?